routes/api.go

```diff
@@ -0,0 +1,64 @@
+package routes
+
+import (
+	"database-ms/app/handlers"
+	"database-ms/app/middleware"
+	sensorRepo "database-ms/app/repositories/sensor"
+	sensorSrv "database-ms/app/services/sensor"
+	"database-ms/config"
+	"database-ms/controllers"
+
+	"github.com/gin-gonic/gin"
+	"gopkg.in/mgo.v2"
+)
+
+var (
+	DbRoute = "/database"
+)
+
+func InitializeRoutes(c *gin.Engine, dbSession *mgo.Session, conf *config.Configuration) {
+	sensorRepository := sensorRepo.New(dbSession, conf)
+	sensorService := sensorSrv.New(sensorRepository)
+	sensorAPI := handlers.NewSensorAPI(sensorService)
+
+	// Routes
+	// TODO: Create middle ware for just token, just key, or both
+	publicEndpoints := c.Group("/database")
+	{
+		// Organization
+		publicEndpoints.GET("/organizations", controllers.GetOrganizations)
+
+		// Sensor
+		// TODO move later as private endpoint
```

Comment:
OK. You created your repo->service->etc model, but what benefits did it really give you? Instead of controllers.CreateSensor, you have sensorAPI.Create. What is worth it? Just some name changing would have solved your problems. I like the idea of separating request types into a separate file, but separating the logic of just getting some sensors into multiple chains of logic seems overkill. I personally think that every line of code added is +1, every line deleted is +2. There are no bonus points for extra code, I give bonus for less code that is clear, concise, and to the point.